.github/workflows/certora_recovery.yml

```diff
@@ -0,0 +1,44 @@
+name: certora
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+  workflow_dispatch:
+
+jobs:
+  verify:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        rule: ['verifyRecoveryModule.sh']
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install python
+        uses: actions/setup-python@v4
+        with: { python-version: 3.11 }
+
+      - name: Install certora cli
+        run: pip install -Iv certora-cli==6.1.3
```

Comment:
Can we follow what we do in the `safe-locking` repo here? Namely, we should have a `requirements.txt` file and use `pip install -r certora/requirements.txt`. This is the standard practice for Python related things:

https://github.com/safe-global/safe-locking/blob/5a8c080dc20a6a15a0c390f4a8056a39c97e0087/.github/workflows/certora.yml#L33

Also, Certora CLI is already on v7, so we should use the latest version of 7.3.0

Reply:
Updated 👍🏾